build.bash

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
+CURRENTDIR=pwd
 if [[ ! -f /usr/include/sqlite3.h ]]
 then
-    curl https://sqlite.org/2021/sqlite-autoconf-3340100.tar.gz -o "/usr/include/sqlite.tar.gz"
-    tar -zxvf /usr/include/sqlite.tar.gz /usr/include/sqlite/
+    cd /usr/include
+    touch sqlite.tar.gz
+    mkdir sqlite
+    curl https://sqlite.org/2021/sqlite-autoconf-3340100.tar.gz -o sqlite.tar.gz
```

Comment:
tsooo xD

pobierasz tutaj paczkę z kodem źródłowym i makefilem i rozpakowujesz to do /usr/include

Reply:
A nie mogę?

Reply:
No nie. To nie jest header-only libka więc musi być osobno zbudowana (chociażby za pomocą tego makefile)

Reply:
Ale jak Ty chcesz dołączyć za pomocą `#include` binarkę? Bo przecież po zbudowaniu powstanie .lib lub jakiś inny podobny plik. Kompilator Ci nie skompiluje tego, co nie jest C++ w plikach .cpp .cc .cxx .c++ czy jakichś tam innych (chyba, że dasz `extern "C"` lub `asm`). Nawet jeśli zasygnalizujesz kompilatorowi, że w tym pliku nie ma C++, to i tak w tym pliku będzie trochę C++, a poza tym, opcja `-x`, może przyjmować wartości **tylko** c, cpp, asm.

Reply:
Słyszałesz o linkerze i fladze `-l` w gcc?

Reply:
Racja, nie pomyślałem o tym, przepraszam.